src/plugins/intel_cpu/tests/functional/single_layer_tests/matmul.cpp

```diff
@@ -759,7 +761,7 @@ const auto fullyConnectedParams2D_Brgemm_nightly = ::testing::Combine(::testing:
 const auto testParams2D_Brgemm_nightly = ::testing::Combine(fullyConnectedParams2D_Brgemm_nightly,
                                              ::testing::Values(MatMulNodeType::FullyConnected),
                                              ::testing::ValuesIn(fusingParamsSet2D_nightly),
-                                             ::testing::ValuesIn(filterSpecificParams_Brgemm()));
+                                             ::testing::ValuesIn(filterSpecificParams_Brgemm(true)));
```

Comment:
In current file only FC case is covered with brgemm_avx2?  Can we extend on matmul case as well?

Reply:
Same as above. Matmul brgemm can support INT8 inference on avx2_vnni, but can't support fp32 inference on avx2 ISA. That is the onednn limitation.